unit_tests/qfdmo/test_acteur_methods.py

```diff
@@ -0,0 +1,28 @@
+"""Test file dedicated to acteur methods"""
+
+import json
+
+import pytest
+
+from unit_tests.qfdmo.acteur_factory import (
+    ActeurFactory,
+)
+
+
+@pytest.mark.django_db
+class TestActeurMethods:
+
+    @pytest.mark.parametrize(
+        "initial,expected",
+        [
+            (None, [{"message": "test"}]),
+            ("  ", [{"message": "test"}]),
+            ("foo", [{"message": "foo"}, {"message": "test"}]),
+            ('[{"message": "bar"}]', [{"message": "bar"}, {"message": "test"}]),
```

Comment:
cf commentaire plus haut

```suggestion
            ('{"message": "bar"}', [{"message": "bar"}, {"message": "test"}]),
```